config/application.rb

```diff
@@ -45,6 +45,10 @@ class Application < Rails::Application
     # Don't generate system test files.
     config.generators.system_tests = nil
 
+    config.hosts << 'atc.library.columbia.edu:3000'
+    config.hosts << 'atc-dev.library.columbia.edu:3000'
+    config.hosts << 'atc-test.library.columbia.edu:3000'
```

Comment:
These lines are good for local login debugging, but I think we'd be better off not committing them to the repository (in case other developers don't run their local copy of the app on port 3000, which I myself sometimes don't when running multiple local apps in parallel).  Easy enough to add locally in the future as needed though.

Reply:
lines removed.